code/game/objects/items/RPD.dm

```diff
@@ -235,6 +237,21 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 /obj/item/pipe_dispenser/attack_self(mob/user)
 	ui_interact(user)
 
+/obj/item/pipe_dispenser/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/rpd_upgrade))
+		install_upgrade(W, user)
+		return TRUE
+	return ..()
+
+/// Installs an upgrade into the RPD checking if it is already installed
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-proc

Reply:
```suggestion
/**
  * Installs an upgrade into the RPD
  *
  *  Installs an upgrade into the RPD checking if it is already installed
  * Arguments:
  * * rpd_up - RPD upgrade
  * * user - mob that use upgrade on RPD
  */
```

Reply:
@ExcessiveUseOfCobblestone Am I do it wrong or do you need something else?